python/ray/serve/tests/test_callback.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+import requests
+import starlette
+import sys
+
+from ray import serve
+
+
+def test_callback(serve_instance, capsys):
```

Comment:
please add tests cases for each where:

1. the import paths are invalid
2. the callbacks raise an exception

Reply:
also for HTTP proxy, add a test case with a bad return type (not middlewares)